widgets/nuxeo-dialog.html

```diff
@@ -98,7 +98,10 @@
           if (!this._instance) {
             const template = Polymer.dom(this).querySelector('nuxeo-dialog > template');
             if (template) {
-              this.templatize(template);
+              // prevent "A <template> can only be templatized once" error
```

Comment:
iirc we're doing `delete template.__templatizeOwner` in nuxeo-slot for this same reason